projects/common/src/lib/controls/refresh-select/refresh-select.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, EventEmitter, Input, OnChanges, OnInit, Output } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { IoTEnsembleState } from '../../state/iot-ensemble.state';
+
+@Component({
+  selector: 'lcu-refresh-select',
+  templateUrl: './refresh-select.component.html',
+  styleUrls: ['./refresh-select.component.scss']
+})
+export class RefreshSelectComponent implements OnInit, OnChanges {
+
+  /**
+   * Incoming State
+   */
+  @Input('state') State: IoTEnsembleState;
```

Comment:
In order to create similarity and consistency within our different languages (C#, TS), use a line break between attributes and properties. In addition, always explicitly type things as public instead of inferring it as is default in js:
@Input('state')
public State: IoTEnsembleState;

Reply:
i see you did explicitly type some things, so just update the ones that aren't